Tools/CISupport/ews-build/steps.py

```diff
@@ -2776,13 +2777,17 @@ class RevertAppliedChanges(steps.ShellSequence):
     flunkOnFailure = True
     haltOnFailure = True
 
-    def __init__(self, **kwargs):
+    def __init__(self, exclude=None, **kwargs):
```

Comment:
Nit: Can consider adding unit-test in `TestRevertAppliedChanges` for exclude. Not a blocker though.